docs/flask.rst

```diff
@@ -154,7 +154,7 @@ For this we will update ``hello_flask.py``.
 
   @APP.route('/hello/<name>/')
   def hello(name):
-      """ Displays the page greats who ever comes to visit it.
+      """ Displays the page. Greets whoever comes to visit it.
```

Comment:
"Displays the page and greets whoever comes to visit"

I believe this would be a tad bit better.